torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -292,15 +290,21 @@ def precompile(self, warm_cache_only_with_cc=None):
                         continue
                     seen_configs.add(new_config)
                     self.launchers.append(
-                        self._precompile_config(new_config, warm_cache_only_with_cc)[1]
+                        self._precompile_config(new_config, warm_cache_only)[1]
                     )
             self.configs = None
 
-    def _precompile_config(self, cfg: Config, warm_cache_only_with_cc: Optional[int]):
+    def get_device_interface(self):
+        # this code cannot run in compile workers, because it imports from torch
```

Comment:
I wonder if there's a way we could assert that?

Reply:
There are asserts in the `verify()` function in the followup PR.